extensions/amp-next-page/0.1/amp-next-page.js

```diff
@@ -86,11 +86,9 @@ export class AmpNextPage extends AMP.BaseElement {
     user().assert(separatorElements.length <= 1,
         `${TAG} should contain at most one <div separator> child`);
 
-    let separator;
+    let separator = null;
```

Comment:
nit: we normally don't initialize to null as undefined works fine in most cases.